src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return sum - 20;
+  } else if (days === 7) {
+    return sum - 50;
+  }
```

Comment:
The use of 'else if' after a 'return' statement is unnecessary. You can remove the 'else' and just have an 'if' statement since the 'return' statement will exit the function if the condition is met.